programmers/난이도별/level01.이상한_문자_만들기/HyeonJeong.py

```diff
@@ -0,0 +1,11 @@
+def solution(s):
+    answer = ""
+    for c in range(len(s)):
+        if s[c] == " " :
+            answer += " "
+        else:
+            if c % 2 == 0 :
+                answer += s[c].upper()
+            else:
+                answer += s[c].lower()
```

Comment:
```suggestion
    for c in s:
        if c == " " :
            answer += " "
        else:
            answer += c.upper() if 소문자일경우 else c.lower()
```
일단 이런식으로 변경해도 될 것 같습니다!

`소문자일경우` 라는 조건을 만족하기 위한 로직을 만들어야할 것 같아요~